Python/Blood Alcohol Content/Blood_Alcohol_Content.py

```diff
@@ -0,0 +1,36 @@
+class BAC(object):
```

Comment:
We need docstrings, Class Docstrings and `__inti__` is not mandatory, but for the rest of the methods, docstrings are a must. Update accordingly.